index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formula 1 Get Together Conference</title>
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="hamburger-logo">
+    
+            <div class="header-container">
+                <img src="images/mobile_menu1.svg" class="ham">
+                <nav>
+                <img src="images/F1_logo.png" class="logo">
```

Comment:
[REQUIRED] Kindly ensure that your images contain alt attributes so that your design could be more accessible.

Reply:
Introduced alt to the images